packages/fluentui/react-bindings/src/hooks/useTriggerElement.tsx

```diff
@@ -6,7 +6,7 @@ import { childrenExist } from '../utils/childrenExist';
 
 type UseTriggerElementOptions = {
   children?: React.ReactNode;
-  trigger?: React.ReactElement;
+  trigger?: React.ReactElement | React.ReactNode | null;
```

Comment:
`React.Node` already covers the other types

```suggestion
  trigger?: React.ReactNode;
```

Reply:
This should be `React.ReactElement`, I will check and fix it.

Reply:
Fixed